src/components v2/Atoms/Forms/InputText/InputText.stories.js

```diff
@@ -0,0 +1,174 @@
+import PLabel from '@/components v2/Atoms/Forms/Label';
+import PInputText from '@/components v2/Atoms/Forms/InputText';
+import PButton from '@/components v2/Atoms/Components/Button';
+import { PText } from '@/components/Typography';
+
+export default {
+  title: 'v2/Atoms/Forms/InputText',
+  component: PInputText,
+  argTypes: {},
+};
+
+const Template = (args, { argTypes }) => ({
+  components: { PInputText, PLabel, PButton, PText },
+  props: Object.keys(argTypes),
+  template: `
+    <div>
+      <p-input-text
+        v-bind='$props'
+        :errors="errs"
+        v-model="name"
+      />            
+      <p-text xs class="ph-mb-5">Name: {{ name }}</p-text>
+      
+    </div>
+  `,
+  data() {
+    return {
+      name: null,
+      errs: [],
+    };
+  },
+  methods: {
+    showError() {
+      if (this.errs?.length) {
+        this.errs = [];
+      } else {
+        this.errs = ['Inline validation error'];
+      }
+    },
+  },
+});
+const LeftIconTemplate = (args, { argTypes }) => ({
+  components: { PInputText, PLabel, PButton, PText },
+  props: Object.keys(argTypes),
+  template: `
+    <div>
+      <p-input-text
+        v-bind='$props'
+        :errors="errs"
+        v-model="name"
+        currency
+        icon-left='Dollar'
+      />            
+      <p-text xs class="ph-mb-5">Value: {{ name }}</p-text>
+      
+    </div>
+  `,
+  data() {
+    return {
+      name: null,
+      errs: [],
+    };
+  },
+  methods: {
+    showError() {
+      if (this.errs?.length) {
+        this.errs = [];
+      } else {
+        this.errs = ['Inline validation error'];
+      }
+    },
+  },
+});
+const RightIconTemplate = (args, { argTypes }) => ({
```

Comment:
**tip:** you can create one template and just pass the props

 `iconRight` can be passed into the args property.
 
 Increases re-usability.